nodejs/week1/avg.js

```diff
@@ -0,0 +1,16 @@
+const args = process.argv.slice(2);
+
+if (args.length === 0) {
+    console.log("No numbers provided. Please provide numbers as arguments.");
+} else {
+    const numbers = args.map(Number);
+
+    if (numbers.some(isNaN)) {
```

Comment:
all is looking good! I would suggest though looking into early return pattern to avoid nesting and else statements for legibility of the code: https://medium.com/swlh/return-early-pattern-3d18a41bba8